nixnet/database/_frame.py

```diff
@@ -177,7 +179,8 @@ def dbc_attributes(self):
         # type: () -> _dbc_attributes.DbcAttributeCollection
         """:any:`DbcAttributeCollection`: Access the frame's DBC attributes."""
         if self._dbc_attributes is None:
-            self._dbc_attributes = _dbc_attributes.DbcAttributeCollection(self._handle)
+            self._dbc_attributes = _dbc_attributes.DbcAttributeCollection(
```

Comment:
No need to make this change.

Reply:
Yeah whoops, must have missed that this got redone in the merge